core/widgets.py

```diff
@@ -0,0 +1,44 @@
+from typing import Any, Dict, Optional
+from django.forms import Widget
+
+class TagsEditor(Widget):
+    input_type = 'hidden'
+    template_name = 'core/widget_tag_editor.html'
+
+    @staticmethod
+    def __unpack_tag(tag):
+        return {'name': tag.name, 'color': tag.color}
+
+    def format_value(self, value: Any) -> Optional[str]:
```

Comment:
Just a comment -- this is the first time I have seen typing used. I'm definitely a big fan of this other languages and would like to refactor Baby Buddy to use it eventually.

Reply:
VS Code is inserting them manually for me. And in my 9-5, we are using them pretty much everywhere. Very handy, specially when combining with an IDE that understands the type annotations so you get your handy Ctrl+Space auto completion! `Any` is a bit of a disappointing annotation here correct in this case, from what I was able to derive.

Anyway, I would add them as-you-go to the project. Slowly they will build up and become helpful. IMO. Anyway, I hope you do not mind that I add them?

Reply:
No problem -- add away! I'll probably start adding them myself after this gets merged in and I get the hang of it.